lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MarkdownUtils.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat Inc. and others.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.lemminx.maven;
+
+public class MarkdownUtils {
+	static final String LINE_BREAK = "\n\n";
+
+	private MarkdownUtils() {}
+	
+	public static String getLineBreak(boolean supportsMarkdown) {
```

Comment:
I don't think it fully belongs here.
Think for example of a case where we intentionally send HTML in case client is LSP4E because we know it works, then this methods becomes more a switch and wouldn't be only about markdown.
But it's not important and let's keep it at it for now. No need to over-engineer that now.